index.js

```diff
@@ -6,15 +6,15 @@ module.exports = (robot, _, Settings = require('./lib/settings')) => {
     const payload = context.payload
     const defaultBranch = payload.ref === 'refs/heads/' + payload.repository.default_branch
 
-    const config = await getConfig(context, 'settings.yml', {}, { arrayMerge: mergeArrayByName })
+    if (!defaultBranch) {
+      // Not the defualt branch, nothing to see here!
```

Comment:
Im not sure how to print with debug logs in probot :( but this should be a debug log